extensions/whox.md

```diff
@@ -72,6 +77,7 @@ are used:
 - `255.255.255.255` is used for `[ip]` when the server refuses to disclose the
   IP address
 - `0` is used for `[account]` when the user is logged out
+- `n/a` is used for `[oplevel]` when the server doesn't support op levels
```

Comment:
In the wild, for example on libera.chat: "n/a" actually means the user is not an op, while "999" means the user is an op. Probably best to match a common implementation in the spec.

Reply:
FWIW, this is unsupported in Ergo, and we always send `0`:

https://github.com/ergochat/ergo/blob/v2.8.0/irc/handlers.go#L3311-L3314

Reply:
Hm. I'd prefer to avoid adding anything in the spec explaining what the values mean, since implementations behave differently. This bullet point just suggests servers that don't have a concept of "op level" to send "n/a". Would it be better to send "0" instead like Ergo does? Or "*" as a completely new value so that clients can tell when the value is completely meaningless or not?

Reply:
I'm happy to not go into the recommended value here. But if we are, it should probably reflect reality.

Reply:
I'd suggest testing on more networks (or browsing their source codes) to see whether they match the solanum/charybdis behaviour and if they do, go with that. But given we're saying it's not reliable, new implementations can effectively put whatever they want here, and clients shouldn't break.

Reply:
Maybe delete this as a normative recommendation and then add something like the following to the next paragraph?

>Servers that do not support reporting channel op levels MUST send a syntactically valid, implementation-defined placeholder value if it is requested, for example `*` or `0`.

Reply:
The problem with a ```0``` value is ```0``` is a valid oplevel, it is in fact the highest oplevel, ```999``` being the lowest (ircu is backwards in that regard), so any integer between 0 and 999 inclusive is indicative of being opped in the channel.

So in my opinion the best choices would be ```*``` or ```n/a````

Reply:
Fair enough, a normative recommendation of `*` (whether or not it matches existing in-the-wild implementations) seems good also.

Reply:
- Unrealircd uses `999` for op and `n/a` as a fallback: https://github.com/unrealircd/unrealircd/blob/2afc57aa38f7d78759b646ccb1f9fc2f9ea4cc85/src/modules/whox.c#L817 
- Inspircd uses `0` as a fallback: https://github.com/inspircd/inspircd/blob/6ff8f38a6d55970af1400cc78994e017ca351c39/src/coremods/core_who.cpp#L499
- Hybrid doesn't support WHOX

It doesn't seem like any server uses `*` yet. That makes me a bit uneasy to use it in this spec, because this spec is just supposed to document existing behavior. So I'd prefer to stick with `n/a`.

Reply:
Updated for wording to allow servers to use any placeholder for oplevel.